lib/src/int32.dart

```diff
@@ -54,12 +54,12 @@ class Int32 implements IntX {
   // TODO(rice) - Make this faster by converting several digits at once.
   static Int32 parseRadix(String s, int radix) {
     _validateRadix(radix);
-    Int32 x = ZERO;
-    for (int i = 0; i < s.length; i++) {
-      int c = s.codeUnitAt(i);
-      int digit = _decodeDigit(c);
+    var x = ZERO;
```

Comment:
nit: I'd be _tempted_ to leave just this one. It's relatively obvious given the constant but maybe a good reminder to the reader given it's the odd one out here and line 62. Not the end of the world given every other `var` in this file looks fine to me.